docs/hive-integration/README.md

```diff
@@ -160,10 +160,17 @@ In case that the Bare Metal Operator is installed, the Baremetal Agent Controlle
 
 - Find the right pairs of BMH/Agent using their MAC addresses
 - Set the Image.URL in the BMH copying it from the InfraEnv's status.
```

Comment:
Is this even still true since converged?

Reply:
(unrelated to PR)

Reply:
No, but there's a specific section for converged.